app/assets/stylesheets/users.scss

```diff
@@ -305,7 +305,10 @@
     }
   }
 }
-
+span i {
```

Comment:
It's better to add another class to the element and apply styles to that class instead of applying to element types. (So that it doesnt conflict later if we have somethinng else that is affected by this style). What do you think?

Reply:
Yaa  right,  @gr455, and  Thanks for reviewing, I will commit the changes soon